src/main/java/com/synopsys/integration/alert/web/model/configuration/FieldModel.java

```diff
@@ -72,10 +70,6 @@ public String getContext() {
         return getField(key).flatMap(fieldValueModel -> fieldValueModel.getValue());
     }
 
-    public Collection<String> getFieldValues(final String key) {
```

Comment:
Why are you removing this?

Reply:
It isn't used anywhere so I removed it. I can put it back in case we use it in the future I guess